docs/modules/servers/pages/local/conf/index.adoc

```diff
@@ -1,23 +1,99 @@
 = James Local Mail Server Configuration
 :navtitle: Configuration
 
+== Configuration by feature
+
+ * Domain names &mdash; provide the default domain names that ????
+ * Database (james-database.properties)
+ * LMTP (lmtpserver.xml)
+ * POP3 (pop3server.xml)
+ * DNS (dnsservice.xml)
+ * Loggin (logback.xml)
+ * James CLI (jmx.properties)
+ * Email rewriting (recipientrewritetable.xml)
+ * Guice extensions (extensions.properties)
+ * JWT (jwt_publickey)
+ * Mailets (mailetcontainer.xml)
+ * SMTP (smtpserver.xml)
+ * Health checks (healthcheck.properties)
+ * Mail storage (mailrepositorystore.xml)
+ * User storage (usersrepository.xml)
+ * IMAP (imapserver.xml)
+ * Mailbox event listeners (listeners.xml)
+ * Sieve filters (managesieveserver.xml)
+ * Web admin (webadmin.properties)
+
+== Configuration file index
+
  * Domain names (domainlist.xml)
  * Database (james-database.properties)
  * LMTP (lmtpserver.xml)
  * POP3 (pop3server.xml)
  * DNS (dnsservice.xml)
  * Loggin (logback.xml)
- * ??? (jmx.properties)
- * ??? (recipientrewritetable.xml)
- * ??? (extensions.properties)
+ * James CLI (jmx.properties)
+ * Email rewriting (recipientrewritetable.xml)
+ * Guice extensions (extensions.properties)
  * JWT (jwt_publickey)
  * Mailets (mailetcontainer.xml)
  * SMTP (smtpserver.xml)
  * Health checks (healthcheck.properties)
  * Mail storage (mailrepositorystore.xml)
  * User storage (usersrepository.xml)
  * IMAP (imapserver.xml)
- * ??? (listeners.xml)
+ * Mailbox event listeners (listeners.xml)
  * Sieve filters (managesieveserver.xml)
  * Web admin (webadmin.properties)
+
+
+== Notes
+Jmx protocol is used by James cli to interactwith the server. Jmx is known to be insecure, and we have as a project to rewrite the cli for james servers in order to rely on webadmin.
+
+
+
+Scenarios for address rewriting
+
+The following scenarios are examples of how you can use address rewriting:
+
+Group consolidation: Some organizations segment their internal businesses into separate domains that are based on business or technical requirements. This configuration can cause email messages to appear as if they come from separate groups or even separate organizations.
+
+The following example shows how an organization, Contoso, Ltd., can hide its internal subdomains from external recipients:
+
+Outbound messages from the northamerica.contoso.com, europe.contoso.com, and asia.contoso.com domains are rewritten so they appear to originate from a single contoso.com domain. All messages are rewritten as they pass through Edge Transport servers that provide SMTP connectivity between the whole organization and the Internet.
```

Comment:
We speak of a **domain alias** here. I would more formulate it:

 - External email can send emails to northamerica.contoso.com, europe.contoso.com, ...
 - Internal users can send email from northamerica.contoso.com, europe.contoso.com, ...

Reply:
Added as comment for now